sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantBicepsNormativeAnnexTest.java

```diff
@@ -42,6 +42,10 @@ void setUp() {
         this.messageStorage = injector.getInstance(MessageStorage.class);
     }
 
+    // NOTE: No PreCondition can ensure that there are Localized texts with a @Ref attribute. This is because
+    //       a device either provides a LocalizationService, in which case all Localized texts will have a @Ref attribute,
+    //       or it doesn't, in which case no Localized text will have a @Ref attribute.
+    //       When a device does not provide a LocalizationService, then this test should not be applied to it.
```

Comment:
This is definitely not the right place to "inform" anyone when not to apply the test, no user will ever read it.

Reply:
Made sure this is also stated in RQM.